datasets/iq/aml_list/crawler.py

```diff
@@ -0,0 +1,135 @@
+import re
+from openpyxl import load_workbook
+from rigour.mime.types import XLSX
+from typing import Optional
+
+from zavod import Context, helpers as h
+
+SOURCE_URLS = ["https://aml.iq/?page_id=2169", "https://aml.iq/?page_id=2171"]
+
+LOC_PERSON_LISTS = [
+    "2017",
+    "2018",
+    "2019",
+    "2020",
+    "2021",
+    "2022",
+    "2023",
+    "2024",
+    "2025",
+]
+LOC_ENT_LIST = ["الكيانات"]
+LOC_IGNORE_LIST = ["الافراد", "القوائم المحلية "]
+INT_LISTS = ["كيانات", "افراد"]
```

Comment:
it'd be nice to have translations of these in comments for quick reference

Reply:
and ENTITY_NAME_REASON. If I failed to do this in the past, I now apologise sincerely :)